grafana/provisioning/dashboards/metrics.json

```diff
@@ -907,11 +905,62 @@
             ],
             "title": "Resources",
             "type": "row"
+        },
+        {
+            "collapsed": true,
+            "gridPos": {
+                "h": 1,
+                "w": 24,
+                "x": 0,
+                "y": 3
+            },
+            "id": 18,
+            "panels": [
+                {
+                    "datasource": {
+                        "type": "loki",
+                        "uid": "${logs_ds}"
+                    },
+                    "gridPos": {
+                        "h": 8,
+                        "w": 24,
+                        "x": 0,
+                        "y": 1
+                    },
+                    "id": 17,
+                    "options": {
+                        "dedupStrategy": "none",
+                        "enableLogDetails": false,
```

Comment:
I'd enable it. Maybe now it doesn't have much useful info, but some day it might be usable.